viewer/src/public/RevealManager.ts

```diff
@@ -89,6 +94,10 @@ export class RevealManager extends RevealManagerBase<CdfModelIdentifier> {
     }
   }
 
+  public getLoadingStateObserver(): Observable<boolean> {
+    return this.loadingStateObserver;
+  }
```

Comment:
I don't think we should expose an`Observable` here as it makes RxJS 'leak out' of Reveal. We should either just have a simple `isLoading`-field that must be polled, or maybe events `loadingStarted`+ `loadingFinished` or `loadingStateChanged`.

Reply:
Yes, I was also thinking about that... Would something like that be fine?

```js
reveal.on('loadingStateChanged', (isLoading) => /* ... */ )
```

@larsmoa @j-bjorne

Reply:
I completely disagree. Why would we have something that the user must create their custom polling on. I would much rather they add an event callback on something that gets triggered, but anything else just sounds like a step back. Also rx is not exposed in the migration. Also this doesn't expose rx as we would add the same kind of callbacks to reveal manager

Reply:
`RevealManager` and friends must be considered public, and they therefore should not expose RxJS. 

I agree that polling probably is a bad idea.

Reply:
Yes, they aren't at this point. Rxjs is not being exposed to the public with this solution :)